sources/dynamodb/README.md

```diff
@@ -32,16 +32,22 @@ To use the tool on a DynamoDB database (it will migrate all tables),
 run
 
 ```sh
-harbourbridge -driver=dynamodb
+harbourbridge schema -source=dynamodb
```

Comment:
This will just do schema conversion?
However the description in line 31 indicates ("migrate all tables") indicates full migration and not just schema migration?

Does it make sense to use the eval subcommand here?

BTW: while thinking about the README, I'm having second thoughts about 'eval' as a subcommand name. It kind of sounds like some kind of experimental mode for HarbourBridge. I'm wondering if the long-winded schema-and-data might be better/self-documenting? What do you think?

Reply:
I have fixed the text to say schema migration and have later provided an example of running with oneshot subcommand. As I am reading the documentation now, oneshot seems to make sense. Let me know your thoughts.